EIPS/eip-6269.md

```diff
@@ -0,0 +1,58 @@
+---
+eip: 6269
+title: Full EVM Equivalence
+description: Canonicalise the definition of Full EVM Equivalence
+author: Pascal Caversaccio (@pcaversaccio)
+discussions-to: https://ethereum-magicians.org/t/evm-equivalence-and-ethereum-stack-compatibility-definition-future-informational-eip/10044
+status: Draft
+type: Meta
+created: 2023-01-06
+---
+
+## Abstract
+
+This EIP standardises the definition of "Full Ethereum Virtual Machine (EVM) Equivalence". Full EVM equivalence is **complete** compliance with the latest version of the [Execution Layer Specification](https://github.com/ethereum/execution-specs/tree/24de2192e02bba11e61746aa9dee04d4e7a8b62e). This definition applies retroactively from the first release of the Ethereum Yellow Paper in April 2014.
```

Comment:
Is there a certain reason we refer to a specific commit @Pandapip1? I would prefer referring to https://github.com/ethereum/execution-specs.

Reply:
Yes, as per EIP-5757 external links to any repository must use a specific commit.

Reply:
But there cannot be a unique ID for that link since it should point always to the latest commit. I trust your judgement here how to deal with it but otherwise people will always get a link of an outdated version...

Reply:
Yes, that's the point. They get a link to a thing that never changes.

Reply:
ok understood - I fixed all the links accordingly and even added `requires: 5757` since we need it in this context.